ppg-server/ppg-server.spec

```diff
@@ -0,0 +1,30 @@
+%global sname   percona-ppg-server
+%global pgmajorversion 14
+%global version 4
+
+Summary:        Base selection of PostgreSQL%{pgmajorversion} components
+Name:           %{sname}-%{pgmajorversion}
+Version:        %{version}
+Release:        1%{?dist}
+Epoch:          1
+License:        PostgreSQL
+Group:          Applications/Databases
+URL:            https://www.percona.com/software/postgresql-distribution
+Packager:       Percona Development Team <https://jira.percona.com>
+Vendor:         Percona, LLC
+BuildArch:      noarch
+
+Requires:       percona-postgresql%{pgmajorversion}-server
```

Comment:
The component versions should specify versions as well to avoid conflicts or accidental upgrades across versions.

Reply:
Made an adjustment, is that how you would like to have it?